traffic_portal/app/src/common/modules/form/cacheGroup/form.cacheGroup.tpl.html

```diff
@@ -128,10 +128,29 @@
                     <input type="checkbox" name="GeoEnabled" ng-model="localizationMethods.GEO"> Geo-IP Database</input>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(cacheGroupForm.fallbacks), 'has-feedback': hasError(cacheGroupForm.fallbacks)}" ng-hide="!isEdgeLoc(cacheGroup.typeId)">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Failover Locations</label>
```

Comment:
You seem to be mixing the terms "Failover _Location_" and "Failover _Cache Group_". This should really just use "Cache Group" everywhere (including in the documentation) because the user is selecting a Cache Group and Physical Locations are another object entirely which could cause confusion.

Reply:
done